vspace/src/main/java/edu/asu/diging/vspace/core/services/impl/SequenceManager.java

```diff
@@ -52,4 +52,9 @@ public ISequence getSequence(String sequenceId) {
         }
         return null;
     }
+
+    @Override
+    public void updateSequence(Sequence sequence) {
```

Comment:
the interface should be used here. otherwise why would you cast in the next line?